.github/workflows/helpers/map-strategy.js

```diff
@@ -0,0 +1,19 @@
+const inputString = process.argv[2];
+
+let outputString;
+
+switch (inputString) {
+  case "major":
+    outputString = "major";
+    break;
+  case "minor":
+    outputString = "feature";
+    break;
+  case "patch":
+    outputString = "bug";
+    break;
+  default:
+    outputString = "unknown";
+}
+
+console.log(outputString);
```

Comment:
**Suggestion:** Instead of having this file, I think we should simply have the same values in the dropdown - so that we don't need to map anything in a js script.

Think that this workflow will be reused in other places, so the simpler it is the better it is to reuse 🥳

Reply:
Note: we can use another action that maps 1:1 as discussed

Reply:
Done! But sorry about the huge commit. I was continuously testing on a fork branch, so I just moved over all the changes at once 😬